src/dswx_sar/initial_threshold.py

```diff
@@ -419,7 +422,11 @@ def tile_selection_wbd(self,
                                 and water_area_sublock_flag \
                                 and validnum > num_pixel_max:
 
-                                if selection_method in ['twele', 'combined']:
+                                # Initially set flag as True
+                                tile_selected_flag = True
+
+                                if any(item in ['twele', 'combined']
+                                       for item in selection_method):
```

Comment:
I thinks set operations are a bit more straightforward.
```suggestion
                                if {'twele', 'combined'}.intersection(set(selection_method)):
```